apps/glusterfs/app_volume.go

```diff
@@ -95,6 +95,18 @@ func (a *App) VolumeCreate(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	for _, volumeOption := range msg.Options {
+		switch volumeOption {
+		case "encryption":
```

Comment:
I fully support a blanket pass-through of volume options with no whitelist. We should allow advanced users who know what they're doing to exercise their judgment. Then in `createVolumeOptionsCommand()` you can just generate a series of `gluster volume set` commands.

I will say I also like the idea of having convenience names for more common combinations of options. So for this I'd suggest still keeping the switch statement to watch for specific keywords, but have the default case be the direct pass-through.